super_team_picker_4/public/JS/shuffle.js

```diff
@@ -0,0 +1,61 @@
+function shuffle(list){
+  for(let i = 0; i < list.length; i++){
+    const randomNum = Math.floor((Math.random() * list.length))
+    const temp = list[randomNum]
+    list[randomNum] = list[i]
+    list[i] = temp
+  }
+}
+
+function teamCount(quantity, names) {
+  shuffle(names)
+  //make empty array of n empty subarray
+  let nameList = []
+  for(let i = 0; i < quantity; i++){
+    nameList.push([])
+  }
+  //loop through all names and add name one by one into each subarray
+  let index = 0
+  for(let i = 0; i < names.length; i++){
+    if(index === quantity){
+      index = 0
+    }
+    nameList[index].push(names[i])
+    index++
+  }
+  return nameList
+}
+
+
+
+//inputs: quantity = num per team
```

Comment:
this should be a proper way to serve what you are trying to do with the comment https://jsdoc.app/tags-param.html